### DISCUSSION TITLE
Welcome to NotrinosERP Discussions!

### DISCUSSION THREAD

**Discussion by notrinos** [notrinos/NotrinosERP#1] [General]
📢 **Announcement** NotrinosERP discussions now moved to http://forums.notrinos.com

**Comment by rudisulistyo** [comment]
themes/canvas/images/edit.svg file not found , there is edit.gif
All file with extension .svg

**Reply by notrinos** [reply]
Will update the missed files soon today.
Thank you @rudisulistyo

**Reply by notrinos** [reply]
Hello @rudisulistyo 
The missing icons have been added to canvas and dropdown theme
The fixed files can be seen on [this commit](https://github.com/notrinos/NotrinosERP/commit/af3df616a4386ad4658b551df6ff7807c87bb378)
